airbyte-integrations/connectors/source-github/source_github/schemas/workflow_jobs.json

```diff
@@ -84,13 +94,16 @@
       "type": ["integer", "null"]
     },
     "runner_name": {
-      "type": ["string", "null"]
+      "type": ["null", "string"]
     },
     "runner_group_id": {
       "type": ["integer", "null"]
     },
     "runner_group_name": {
-      "type": ["string", "null"]
+      "type": ["null", "string"]
+    },
+    "repository": {
+      "type": ["null", "string"]
```

Comment:
Is this nullable? Most `repository` keys don't seem like they are

Reply:
I have changed it to be `non-nullable`